tests/rundb/test_httpdb.py

```diff
@@ -201,6 +203,24 @@ def test_log(create_server):
     assert data == body, "bad log data"
 
 
+@pytest.mark.skipif(
+    sys.platform == "darwin",
+    reason="We are developing on Apple Silicon Macs,"
+    " which will most likely fail this test due to the qume emulator,"
+    " but should pass on native x86 architecture",
+)
+def test_api_boot_speed(create_server):
+    run_times = 50
+    expected_time = 30
+    runs = []
+    for i in range(run_times):
+        start_time = time.perf_counter()
+        create_server()
+        end_time = time.perf_counter()
+        runs.append(end_time - start_time)
+    assert sum(runs) / run_times <= expected_time
```

Comment:
```suggestion
    avg_run_time = sum(runs) / run_times
    assert avg_run_time <= expected_time, "Seems like a performance hit on creating api server"
```

Reply:
better explain _why_ the assertion failed - throw a bone to whoever fails on it
in addition, better save the magic calculations on a side variable to give it a name of what we calculate